client/js/app.js

```diff
@@ -70,8 +70,5 @@ function initWithRaven() {
   }
 }
 
-if (document.readyState !== 'loading') {
-  initWithRaven();
```

Comment:
The readyState check was there deal with the situation of the DOMContentLoaded listener being added after the event had already fired, in which case the javascript would never run.